sqldelight-idea-plugin/src/main/kotlin/com/squareup/sqldelight/intellij/lang/SqlDelightFileViewProviderFactory.kt

```diff
@@ -96,7 +96,15 @@ private class SqlDelightFileViewProvider(
     threadPool.schedule(
       {
         ApplicationManager.getApplication().invokeLater(
-          Runnable { generateSqlDelightCode() },
+          {
+            try {
+              generateSqlDelightCode()
+            } catch (e: Throwable) {
+              // IDE generating code should be best effort - source of truth is always the gradle
+              // build, and its better to ignore the error and try again than crash and require
+              // the IDE restarts.
```

Comment:
Worth an `e.printStackTrace()`? Otherwise it's impossible to even see what's happening should a problem arise.

Reply:
merged without looking at this lol WOOPS

yes i think thats a good idea

Reply:
ill just commit that to master